src/appleseed/renderer/modeling/display/display.cpp

```diff
@@ -118,7 +119,7 @@ void Display::open(const Project& project)
     }
     catch (const ExceptionDictionaryItemNotFound&)
     {
-        RENDERER_LOG_FATAL("%s", "cannot open display: bad parameters.");
+        RENDERER_LOG_FATAL("%s", "cannot open display: missing plugin_name parameter.");
```

Comment:
`RENDERER_LOG_FATAL()` will terminate the application. It should almost never be called, except in command line tools themselves (not in library code).

Change `open()` to return some kind of success/failure result (even a simple bool) and handle errors in the caller.